DocumentsFromSnapshotMigration/build.gradle

```diff
@@ -148,11 +148,27 @@ test {
 }
 
 task slowTest(type: Test) {
+    doFirst {
+        println "Running slow tests..."
+    }
+
     useJUnitPlatform()
     dependsOn(':TrafficCapture:dockerSolution:buildDockerImage_elasticsearchTestConsole')
     jacoco {
         enabled = true
     }
+    testLogging {
+        events "passed", "skipped", "failed"
+        exceptionFormat "full"
+        showExceptions true
+        showCauses true
+        showStackTraces true
+        // showStandardStreams = true
+    }
+    reports {
+        html.required = true
+        html.destination file("${buildDir}/reports/tests/slowTest")
+    }
```

Comment:
For checkin, make sure that this is consistent with our other projects.  I'm not sure currently what it should be, but it's evolved over time.

Reply:
This was pulled from the RFS sub-project; will check what the other sub-projects use though.